crates/volta-core/src/tool/mod.rs

```diff
@@ -192,3 +193,27 @@ fn registry_fetch_error(
     let from_url = from_url.as_ref().to_string();
     || ErrorKind::RegistryFetchError { tool, from_url }
 }
+
+pub fn check_shim_reachable(shim_name: &str) {
+    let home = match volta_home() {
+        Ok(home) => home,
+        Err(_) => return,
+    };
+
+    let shim = home.shim_file(shim_name);
+    let resolved = match which::which(shim_name) {
+        Ok(resolved) => resolved,
+        Err(_) => {
+            info!(
+                "{} cannot find command {}. Please ensure that {} is at the front of your PATH.",
```

Comment:
Apologies for multiple rounds of iteration, but for an error / info message, we try to take special care to be as helpful as possible. Three relatively minor things that jump out here:

* Would it make sense to use the `home.shim_dir()` value for the CTA part? Using the shim path itself would mean that we would suggest putting `~/.volta/bin/node` on the `PATH`, when what we're actually suggesting is having `~/.volta/bin` on the `PATH`
* Since Volta works on Windows as well (and we do these same checks), we should match Windows' casing for `Path` on those systems. So we should have `PATH` on Unix and `Path` on Windows.
* Also related to Windows, with how the Environment variables are handled, I'm not sure "at the front of your Path" makes the most sense. For this case (the command couldn't be found at all), we can probably say something like "Please ensure that {} is available on your Path".